Sources/WebMIDIKit/MIDIPortMap.swift

```diff
@@ -66,8 +66,7 @@ public class MIDIPortMap<Value: MIDIPort>: Collection {
 
     internal final func remove(_ endpoint: MIDIEndpoint) -> Value? {
         // disconnect?
-        guard let port = self[endpoint] else { assert(false); return nil }
-        assert(port.state == .connected)
```

Comment:
@adamnemecek Since nothing bad happens when the state is _not_ `.connected`, removing this wasn't a problem. But I think we could add an `if port.state == connected` path here if cleanup and disconnecting would be required.